peachjam/views/legal_instrument.py

```diff
@@ -10,9 +10,10 @@ class LegalInstrumentListView(FilteredDocumentListView):
     model = LegalInstrument
     template_name = "peachjam/legal_instrument_list.html"
     navbar_link = "legal_instruments"
+    queryset = LegalInstrument.objects.prefetch_related("author", "nature")
 
     def get_queryset(self):
-        queryset = LegalInstrument.objects.prefetch_related("author", "nature")
+        queryset = super(LegalInstrumentListView, self).get_queryset()
         return queryset.order_by("-date")
```

Comment:
If you move the order_by into the `queryset` attrib, then `get_queryset` can fall away completely.